pom.xml

```diff
@@ -157,81 +158,31 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
             </plugin>
+            <plugin>
```

Comment:
The plugin was introduced with https://github.com/DANS-KNAW/easy-ddm/commit/d21dc3b87d6bb4f7095911c83b639a115bdf57bf In fact I'm reverting that change. Intellij shows a more consise diff in both changes.

The profile allowed to choose between plugin configurations with an optional `-DSNAPSHOT_SCHEMA` on the mvn command line. I didn't like to copy-paste such a huge construct for each project using emd or ddm (though we don't yet I agree we need it but that's another discussion) and now discovered supplying `-Deasy.emd.version=...-SNAPSHOT` does the same job but that variable wasn't available back then.

The choice is required during a not yet published (=/= merged) upgrade of EASY-SCHEMA for unit testing with SpecialValidator. Aha: I should update its javadoc (and add the reason to the new readme section) so it was a good question!